modules/consensus/accept_test.go

```diff
@@ -1349,7 +1351,7 @@ func (cst *consensusSetTester) testPaymentChannelBlocks() error {
 		// txn needs to be fully custom. To get a custom txn, manually create an
 		// address and then use the wallet to fund that address.
 		channelSize := types.NewCurrency64(10e3)
-		channelFundingSK, channelFundingPK, err := crypto.GenerateSignatureKeys()
```

Comment:
It's how `crypto/rand` works. There's a `var Rand io.Reader` that provides cryptographic entropy. Calling `rand.Reader.Read()` and `rand.Read()` actually do the same exact thing - `rand.Read()` is just a passthrough function.

I think this style makes sense. There's no reason to force people to call `skg := crypto.NewSKG(); skg.Generate()`, calling `crypto.Generate()` should be sufficient, especially since that's practically speaking going to be 100% of the non-testing use cases.